frontend/finance-tracker/src/components/navigation/nav-user.tsx

```
@@ -0,0 +1,90 @@
+    return (
+        <Avatar>
+            <AvatarImage src="/placeholder.svg?height=40&width=40" />
+            <AvatarFallback className="border">{fallbackText}</AvatarFallback>
+        </Avatar>
+    );
+}
+
+export default function NavUser() {
+    const { data: session } = useSession();
+    const user = session?.user;
+    const avatarFallbacktext = user?.name![0].toUpperCase() ?? "U";
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)